index.ts

```diff
@@ -85,24 +87,31 @@ export interface RemarkMermaidOptions {
    *
    * Set to `null` explicitly to disable this.
    *
+   * **Note**: This options is only supported in Node.js. In the browser this option is unused.
+   *
    * @default defaultSVGOOptions
    */
   svgo?: OptimizeOptions | null;
 
   /**
    * The mermaid options to use.
+   *
+   * **Note**: This options is only supported in Node.js. In the browser this option is unused. If
+   * you this in a browser, call `mermaid.initialize()` manually.
    */
   mermaidOptions?: Parameters<typeof mermaid['initialize']>[0];
 }
 
 /**
  * @param options Options that may be used to tweak the output.
  */
-const remarkMermaid: Plugin<[RemarkMermaidOptions], Root> = ({
-  launchOptions,
-  mermaidOptions = {},
-  svgo = defaultSVGOOptions,
-}) => {
+const remarkMermaid: Plugin<[RemarkMermaidOptions?], Root> = (options) => {
+  if (!options?.launchOptions?.executablePath) {
+    throw new Error('The option `launchOptions.executablePath` is required when using Node.js');
+  }
```

Comment:
Maybe useful to throw in browsers if options are passed

Reply:
I thought about it, but I think it shouldn’t throw then. A user might define these options in a context where their code can run either in the browser or in Node.js. I wouldn’t want the browser version to break then just because the options are ignored.

Reply:
good point!